manifests/init.pp

```diff
@@ -96,22 +98,28 @@
         }
       }
       default: {
-        notify {'influxdb_repo_warn':
+        notify { 'influxdb_repo_warn':
           message  => "Unable to manage repo on ${facts['os']['family']}, using archive source",
           loglevel => 'warn',
         }
       }
     }
 
-    package {'influxdb2':
+    package { 'influxdb2':
       ensure  => $version,
       require => Yumrepo[$repo_name],
       before  => Service['influxdb'],
     }
+
+    service { 'influxdb':
```

Comment:
We made some changes to how the service is managed as part of #35.  It looks like this branch has those commits, but also has the code that was removed in 3741d1f0.  Could you update this so the code matches what's currently in `main`?